src/native/minipal/cpufeatures.c

```diff
@@ -125,6 +155,19 @@ static uint32_t avx512StateSupport()
     return ((_xgetbv(0) & 0xE6) == 0x0E6) ? 1 : 0;
 }
 
+#ifndef XSTATE_MASK_APX
```

Comment:
This was already defined in this file at line 75

Reply:
Line 75 is the definition under Linux, and under windows, I suppose the definition should be from winnt.h, I'm not quite familiar with how PAL works with each part. If so, we will need this definition here until OS supports APX and defines these masks in winnt?

Reply:
Ah, I've missed the fact that one definition was for Windows and the other for Unix. So we can keep it as is until the symbol makes it to the windows SDK and then remove the Windows defines.